README.md

```diff
@@ -27,3 +27,27 @@ The following is a summary of the top level commands.
 - `c d` is for everything Docker.
 - `c dc` is for everything Docker Compose.
 - `c npm` is for everything NPM.
+
+## Configuration
+
+The Idearium cli can be customised through configurations. Configurations are JSON files loaded from a projects `devops` folder.
+
+### NPM configuration
+
+The Idearium cli supports an NPM configuration. The configuration can be used to provide the cli without information about where NPM commands can be run. To provide this information, create a JSON file at `devops/npm.json` with the following:
+
+```
+{
+    "locations": {
+        "name": "./app/root/app/",
+        "project": "./"
+    }
+}
+```
+
+You should customise it, but you need to provide a `locations` key, containing an object with a name and folder for each NPM location in your project. You'll use the name to reference the location.
```

Comment:
An array would make it complex to reference the location and find it though? Also, I like an object because it enforces uniqueness and for the purpose of the NPM locations, uniqueness is required.